pages/download.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <!-- meta tags  -->
+
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+        <!-- favicon  -->
+        <link rel="shortcut icon" href="../images/favicon.png" type="image/x-icon" />
+
+        <!-- fonts -->
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap" rel="stylesheet" />
+
+        <!-- CSS -->
+        <link rel="stylesheet" href="../css/styles.css" />
+
+        <!-- Title -->
+        <title>Git - Downloading Package</title>
+      </head>
+
+    <body>
+        <!-- header section -->
+    <div class="inner-container">
+    <header>
+      <!-- brand logo -->
+      <a href="../index.html">
+        <img src="../images/logo@2x.png" alt="git logo" width="110" height="46" />
+      </a>
+
+      <!-- page heading -->
+      <span id="page-heading">--local-branching-on-the-cheap</span>
+
+      <!-- search bar -->
+      <form class="search">
+        <input type="text/javascript" placeholder="Search entire site..." autocomplete="off" />
+      </form>
+    </header>
+
+    <div class="inner">
+        <div id="content-wrapper">
+          <button class="sidebar-btn"></button>
```

Comment:
remove this button for now as the sidebar functionality cannot be implemented.

Reply:
Done